src/prism.c

```diff
@@ -4037,19 +4037,30 @@ pm_local_variable_or_write_node_create(pm_parser_t *parser, pm_node_t *target, c
     return node;
 }
 
+static bool context_p(pm_parser_t *parser, pm_context_t context);
+
 /**
  * Allocate a new LocalVariableReadNode node.
  */
 static pm_local_variable_read_node_t *
 pm_local_variable_read_node_create(pm_parser_t *parser, const pm_token_t *name, uint32_t depth) {
+    pm_constant_id_t name_id = pm_parser_constant_id_token(parser, name);
+
+    if (
+        context_p(parser, PM_CONTEXT_DEFAULT_PARAMS) &&
```

Comment:
This is going to be checked a lot, I'm wondering if we can make this more efficient by just checking `parser->current_param_name == name_id`, since it will always necessarily be `PM_CONTEXT_DEFAULT_PARAMS` if the value is set?

Reply:
Yes, `0` would work, constants have to be `>= 1`

Reply:
Is a macro or a constant for this value defined?